index.js

```diff
@@ -15,6 +14,9 @@ async function run() {
     const repo = {
       owner: github.context.payload.repository.owner.login,
       repo: github.context.payload.repository.name,
+      // for testing locally with act
+      // owner: 'agrc',
+      // repo: 'get-next-version-action'
```

Comment:
**thought**: should we use an environmental variable in an if statement here?

Reply:
I wonder how that would work with act since it uses containers. Possibly a good enhancement though.